rbd.c

```diff
@@ -1456,7 +1456,11 @@ static int tcmu_rbd_unmap(struct tcmu_device *dev, struct tcmur_cmd *tcmur_cmd,
 	if (ret < 0)
 		goto out_free_aio_cb;
 
+#if LIBRBD_SUPPORTS_WRITE_ZEROES
+	ret = rbd_aio_write_zeroes(state->image, off, len, completion, 0, 0);
```

Comment:
I would just drop this commit -- the "write_zeroes" call would be more appropriate for the `write_same` SCSI command than the discard.

Reply:
Okay, since that workaround could work well.